### DISCUSSION TITLE
Markdown support

### DISCUSSION THREAD

**Discussion by snielsson** [ZarehD/AspNetStatic#16] [Q&A] [answered]
Hi, I am looking for a simple way to create a static blog based on markdown files. How would I do that with AspNetStatic - would I need to install say Markdig and then process the .md files to .html or does AspNetStatic support that out of the box ?

**Comment by ZarehD** [comment, marked as answer]
Hello @snielsson.

Sorry for the long delay in responding. I'm only just home from being hospitalized.

Yes, you would have to use something like Markdig to render the .md files as HTML.

AspNetStatic is in no way involved in how the content for a route is rendered by the server -- you can use whatever tech you want for that, like Markdig;. AspNetStatic simply captures the rendered content and writes it to a static (.html) file.

**Reply by snielsson** [reply]
No worries and thanks for the answer! Sorry to hear about you being hospitalized and I hope you are well.

**Reply by ZarehD** [reply]
Thank you for the kind words. Recovery is a bit slower than I'd like but it's steady.
